lib/headers.js

```diff
@@ -122,6 +229,14 @@ async function getResourceHeaders(url, options) {
 }
 
 module.exports = {
+    getHeaders,
     parseResourceHeaders,
-    getResourceHeaders
+    getResourceHeaders,
+    getContentRange,
+    getContentLength,
+    getContentType,
+    getETag,
+    getFilename,
+    getLastModified,
+    getSize
```

Comment:
I assume these are going to be used in the new pipeline and that's why they are exported (even though they are not all used in this PR)

Reply:
Yes, that's correct.